.github/workflows/teardown-oss-feature-branch.yml

```diff
@@ -0,0 +1,53 @@
+name: Remove Docker Images and Uninstall Helm Release
+on:
+  workflow_dispatch:
+    inputs:
+      version-postfix:
+        description: "Postfix for the default version: 0.0.0-rc. As an example: 0.0.0-rc99"
+        required: true
+        type: string
+
+env:
+  PRERELEASE_TAG: 0.0.0-rc
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_KEY }}
+  AWS_DEFAULT_REGION: "us-east-1"
+
+jobs:
+  remove-images:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set production tag
```

Comment:
you can get rid of the `set production tag` step and also the `log in to dockerhub` step

also rename the `Remove Docker Images` to something like `Delete Kubernetes Namespace`

Reply:
Dont we need to log into Dockerhub in order to delete the namespace?